api/apps/api/test/access-control/project-acl-get.e2e-spec.ts

```diff
@@ -0,0 +1,132 @@
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { getFixtures } from './project-acl.fixtures';
+
+let fixtures: FixtureType<typeof getFixtures>;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+});
+afterEach(async () => {
+  await fixtures?.cleanup();
+});
+
+test(`getting project users when user is not added to project`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  const response = await fixtures.WhenGettingProjectUserAsNotInProject(
+    projectId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
+});
+
+test(`getting project users as the owner and unique user`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  const response = await fixtures.WhenGettingProjectUsersAsOwner(projectId);
+  fixtures.ThenSingleOwnerUserInProjectIsReturned(response);
+});
+
+test(`getting project users with user in project but is viewer`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  await fixtures.WhenAddingANewViewerToTheProjectAsOwner(projectId);
+  const response = await fixtures.WhenGettingProjectUsersAsViewer(projectId);
+  fixtures.ThenForbiddenIsReturned(response);
+});
+
+test(`getting project users with user in project but is contributor`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  await fixtures.WhenAddingANewContributorToTheProjectAsOwner(projectId);
+  const response = await fixtures.WhenGettingProjectUsersAsContributor(
+    projectId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
+});
+
+test(`getting project users with user in project but is other owner`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  await fixtures.WhenAddingANewOwnerToTheProjectAsOwner(projectId);
+  const response = await fixtures.WhenGettingProjectUsersAsOtherOwner(
+    projectId,
+  );
+  fixtures.ThenAllUsersinProjectAfterAddingAnOwnerAreReturned(response);
```

Comment:
I suppose we can skip complicated descriptions and keep `ThenProjectUsersAreAvailable` , as the rest of information is represented by steps.